src/pages/api/users/GET.ts

```diff
@@ -0,0 +1,19 @@
+import connectMongo from "../../../util/connectMongo";
+import User from "../../../../models/user";
+
+/**
+ * @param {import('next').NextApiRequest} req
+ * @param {import('next').NextApiResponse} res
+ */
+
+export default async function getUsers(req: any, res: any) {
+  try {
+    await connectMongo();
+    const users = await User.find();
+    res.json({ users: users });
```

Comment:
WARNING! Unless I'm mistaken, there is a major security problem here. You're leaking your entire database of users, including their hashed passwords. I see no reason to have an API resource like this in this project, but if you do want an endpoint like this, it should at least be restricted to some special type of (or individual) user.

Currently, anyone can just browse to your endpoint and get a list of all the users.